README.md

```diff
@@ -49,8 +49,15 @@ The goal is to use [Bower](http://bower.io) to manage front-end dependencies in
 * ```ng serve```
 
 #### How to deploy with [Github Pages](https://pages.github.com/)?
-* ```git checkout master```
-* ```ng github-pages:deploy --message "Optional commit message"```
+With these very simple steps you can have loklak_search deployed:
+* **Step 1:** Fork loklak_search repository and clone it to your desktop
+* **Step 2:** In you gh-pages branch in `index.html` file change `<base href="/"` to `<base href="/loklak_search/">`
+* **Step 3:** Commit and push your changes
+* **Step 4:** Then checkout to your master branch `git checkout master`
+* **Step 5:** Deploy running this : ```ng github-pages:deploy --message "Optional commit message"```
```

Comment:
i guess just to be safe I will bring replace position of 2 and 3 with 4 & 5, since thats the way I did